src/app/api/schemas.py

```diff
@@ -71,38 +77,62 @@ class TokenPayload(BaseModel):
     scopes: List[str] = []
 
 
-class SiteIn(BaseModel):
+# Location
+class _FlatLocation(BaseModel):
+    lat: float = Field(..., gt=-90, lt=90, example=44.765181)
+    lon: float = Field(..., gt=-180, lt=180, example=4.514880)
+
+
+class Location(_FlatLocation):
+    elevation: float = Field(..., gt=0., lt=10000, example=1582)
+
+
+class DefaultLocation(BaseModel):
```

Comment:
inherit  Location? What is the diff between Location and DefaultLocation?

Reply:
the default values for class attributes